pkg/migrate/migrate.go

```diff
@@ -46,6 +50,18 @@ type Options struct {
 	SkipSourceValidation bool
 }
 
+type pvcCtx struct {
+	claim     *corev1.PersistentVolumeClaim
+	usedByPod *corev1.Pod
+}
+
+func (pvc pvcCtx) getNodeNameRef() string {
+	if pvc.usedByPod == nil {
+		return ""
+	}
+	return pvc.usedByPod.Spec.NodeName
+}
+
```

Comment:
these should go below the exported types/functions, as they're not exported
at least IMO

Reply:
(there's only one exported type and two exported options)

If you want to split this into more files that could be a good idea too

Reply:
Hmmm...I would prefer it to be defined after exported types and before functions as procedurally it makes the file readable for me. Also, there isn't enough types to justify a separate file IMO.

Reply:
lets move this down below the exported functions